lib/platform/bitbucket-server/index.md

```diff
@@ -53,3 +52,9 @@ You should then receive a "Configure Renovate" onboarding PR in any projects tha
 
 We support all Bitbucket Server versions which are not EOL.
 See [Atlassian Support End of Life Policy](https://confluence.atlassian.com/support/atlassian-support-end-of-life-policy-201851003.html#AtlassianSupportEndofLifePolicy-BitbucketServer) for uptodate versions.
+
+## Assignees support
+Assignee support is added in version 17.2.0 of Renovate.
+It's available in all supported Bitbucket versions.
```

Comment:
This sentence is a bit tricky to understand.

What does `all supported BitBucket versions` mean here? Do we mean:

- Supported by us (renovate)?
- Supported by Bitbucket with updates/support?

Reply:
In this case both are the same, but yes can be more clearer